apps/site/assets/css/_footer.scss

```diff
@@ -58,38 +139,303 @@
 // MOBILE FOOTER STYLES
 //
 
-@include media-breakpoint-down(sm) {
+@include media-breakpoint-up(lg) {
   .m-footer__group {
-    display: none;
+    padding-left: 40px;
+  }
+}
+
+address {
+  .m-footer__item {
+    a {
+      color: $brand-primary;
+    }
+
+    @include media-breakpoint-down(xs) {
+      font-size: 14px;
+      line-height: 21px;
+    }
   }
+}
 
-  .m-footer__group-title {
-    display: none;
+.call-us-container {
+  margin-bottom: -50px;
+  background: $brand-primary-lightest-contrast;
+  padding-bottom: 0;
+  padding-top: 22px;
+  flex-basis: 30%;
+  border-radius: 50px 0 0 0;
```

Comment:
Can you add the adjustments for smaller breakpoints? In mobile this should be 24px and in tablet sm and md it's 40px. Same goes for the dark blue container. :)

Reply:
I now see it as 40px but the `border-radius` is not going down to 24px on mobile.